app/grandchallenge/cases/views.py

```diff
@@ -161,6 +173,88 @@ def get_object(self):
         return img
 
 
+class ImageWidgetSelectView(LoginRequiredMixin, View):
+    def get(self, request, *args, **kwargs):
+        query_params = request.GET.copy()
+        interface = query_params.pop("interface", None)[0]
+        widget_name = query_params.pop("WidgetChoice-" + interface, None)[0]
+        help_text = query_params.pop("help_text", None)
+        current_value = query_params.pop("current_value", None)
```

Comment:
Why not access these directly from `request.GET`? Here, if `interface` or `WidgetChoice-` are not present then an type error will be raised as `None` is not sub-scriptable. 

```suggestion
        interface = request.GET["interface"][0]
        widget_name = request.GET[f"WidgetChoice-{interface}"][0]
        help_text = request.GET.get("help_text")
        current_value = request.GET.get("current_value")
```